network/dag/test.go

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (C) 2021. Nuts community
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ *
+ */
+
+package dag
+
+import (
+	"crypto/ecdsa"
+	"crypto/elliptic"
+	"crypto/rand"
+	"encoding/binary"
+	"fmt"
+	"github.com/lestrrat-go/jwx/jwa"
+	"github.com/lestrrat-go/jwx/jws"
+	"github.com/nuts-foundation/nuts-node/crypto/hash"
+	"time"
+)
+
+var privateKey, _ = ecdsa.GenerateKey(elliptic.P256(), rand.Reader)
+
+func CreateTestDocument(num uint32, prevs ...hash.SHA256Hash) Document {
```

Comment:
I think this is still part of the dag namespace since the test.go file does not end in "_test.go".
Also, perhaps add a comment on how to use it?

Reply:
If you postfix it with _test you can't use it from tests outside the package. I'll add a godoc comment on how to use it.